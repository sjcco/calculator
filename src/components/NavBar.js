import { NavLink } from 'react-router-dom';

const NavBar = () => (
  <>
    <nav className="nav-wrapper">
      <div className="container">
        <a href="/" className="brand">Math-Magicians</a>
        <ul className="nav-links">
          <li><NavLink exact to="/">Home</NavLink></li>
          <li><NavLink to="/calculator">Calculator</NavLink></li>
          <li><NavLink to="/quotes">Quote of the day</NavLink></li>
        </ul>
      </div>

    </nav>
  </>
);

export default NavBar;
