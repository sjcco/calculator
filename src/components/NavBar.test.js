import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavBar from './NavBar';

test('render page correctly', () => {
  const component = renderer.create(
    <BrowserRouter>
      <NavBar />
    </BrowserRouter>,
  ).toJSON();
  expect(component).toMatchSnapshot();
});
