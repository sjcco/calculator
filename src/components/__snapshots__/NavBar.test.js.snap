// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render page correctly 1`] = `
<nav
  className="nav-wrapper"
>
  <div
    className="container"
  >
    <a
      className="brand"
      href="/"
    >
      Math-Magicians
    </a>
    <ul
      className="nav-links"
    >
      <li>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          Home
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          href="/quotes"
          onClick={[Function]}
        >
          Quote of the day
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
